libs/api/metadata-converter/src/lib/iso19139/read-parts.ts

```diff
@@ -381,6 +381,7 @@ export function extractDatasetOnlineResources(
     mapArray(
       ([isService, isDownload, protocol, url, name, description, mimeType]) => {
         if (isService) {
+          //FIXME: allow writing identifier only for wms and wfs?
```

Comment:
@jahow This is the only point on which I'm not sure how to fix the converter. As it is currently, for service protocols other than WMS and WFS, the identifier is not lost, but it's transferred to the name property.

This is particularly disturbing when reloading a record after it's been first saved as draft. The attributes aren't kept as they were first inputted.